android-core/src/main/java/com/mparticle/internal/ConfigManager.java

```diff
@@ -51,6 +51,7 @@ public class ConfigManager {
     static final String KEY_OPT_OUT = "oo";
     public static final String KEY_UNHANDLED_EXCEPTIONS = "cue";
     public static final String KEY_PUSH_MESSAGES = "pmk";
+    public static final String DIRECT_URL_ROUTING = "dur";
```

Comment:
Any reason to deviate from the "KEY" nomenclature used for these?

```suggestion
    public static final String KEY_DIRECT_URL_ROUTING = "dur";
```